install_on_windows/tutorial_11-0.md

```diff
@@ -0,0 +1,50 @@
+# Install on Windows
+
+This document collects the different options available on Windows to bring
+Gazebo into a Windows system.
+
+Windows is not a fully officially supported platform by Open Robotics for
+Gazebo. Open Robotics guarantees that the code is compatible with compilations
+on Windows but there are no packages officially available. Since Gazebo is open
+source, the Gazebo Community provides distribution using different projects.
+
+## Community support
+
+The Gazebo Community runs different projects to facilitate the
+Windows installation providing all the dependencies and patch the code to solve
+problems at runtime (help from Open Robotics is usually provided).
+
+For the different Community projects available,
+[checkout the issue tracker](https://github.com/osrf/gazebo/issues/2901). Below
+is a quick overview of some of them.
+
+### Gazebo on Windows via conda-forge
+
+[conda-forge](https://conda-forge.org/) is a collection of packages to be used
+with conda or miniconda installers.
```

Comment:
Minor comment: [conda](https://docs.conda.io/en/latest/) is the package manager, while [miniconda](https://docs.conda.io/en/latest/miniconda.html) that contains packages from the `defaults` channel, that is different from the `conda-forge` channel. Sorry if the situation is so confusing. I think it is better to just refer the conda package manager here:
```suggestion
with the conda package manager.
```